watch-library/shared/watch/watch_private_display.c

```diff
@@ -91,6 +91,27 @@ void watch_display_character(uint8_t character, uint8_t position) {
     if (character == 'T' && position == 1) watch_set_pixel(1, 12); // add descender
     else if (position == 0 && (character == 'B' || character == 'D')) watch_set_pixel(0, 15); // add funky ninth segment
     else if (position == 1 && (character == 'B' || character == 'D' || character == '@')) watch_set_pixel(0, 12); // add funky ninth segment
+    else if (position == 0) {
+        // Clean-up for fancy symbols in the first location.
```

Comment:
This is definitely the wrong thing to do, I just wasn't sure what the right thing to do was.

In particular, I was concerned by this comment:

```
0b11000000, // * (The + sign for use in position 0) 
```

which suggests that some other face is relying on * looking like + in position 0.

I think the best solution is to have an entirely separate Character_Set map for position 0. Thoughts welcome.

(Well, the _real_ best solution would be to calculate appropriate ISDATA things for characters in every position and add some combining logic, I think, as this would make things super fast/easy)

Reply:
My thinking was that you would just emit whatever character lights up the correct segments; unless you need a character that doesn't exist in the character map, there's no need to go this deep into the watch library. 

It looks like you'd use `*` for addition here, and `H` for multiplication. I get where it's confusing, but it solves the 99% case of being able to display something resembling a + at every location, and having a special case for position 0. 

I get that it's a bit confusing for this case in particular, but I'd rather clear that up with a comment in the watch face — Pulsometer is the only one that currently uses a + in this position — than spend additional cycles on a special character set for this one position.

Reply:
Sure, will do. Thanks! It doesn't fit too badly in the code.

With the division one, I prefer using the middle segment (i.e. skewed Z shape, rather than separated bars) - thoughts? Or does it cause other issues?

Reply:
Sorry for the delay in reviewing this. It's not an issue that your watch face wants to display it this way, but I'd rather not code the special cases into `watch_private_display`, which is a lot deeper than it needs to be. It's much better, if your watch face needs to display a particular symbol, to put these pixel writing bits into your watch face. 

Putting them here means every watch face needs to go through this switch statement every time they write a character to position 0, even if the watch face that needs them isn't installed.